hw3/Task_11.ts

```diff
@@ -0,0 +1,16 @@
+//Task 11
+// Написать программу, которая принимает число:
+// 1. Если число делится без остатка на 5 выводим сообщение Fiz
+// 2. Если число делится без остатка на 3 выводим сообшение Buz
+// 3. Если число делится без остатка и на 3 и на 5 выводим сообшение FizBuz
+
+
+let i = 15;
+
+if (i % 5 === 0) {
+    console.log('Fiz');
+} else if (i % 3 === 0) {
+    console.log('Buz');
+} else if (i % 5 === 0 && i % 3 === 0) {
+    console.log('FizBuz');               // не выводится именно это значение
+}
```

Comment:
Try to change sequence of if else blocks

Reply:
Change of sequence helped. Fixed.